PersonalBlogPlatform.Core/AutoMapperProfiles/CommentAddRequestProfile.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoMapper;
+using PersonalBlogPlatform.Core.Domain.Entities;
+using PersonalBlogPlatform.Core.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PersonalBlogPlatform.Core.AutoMapperProfiles
+{
+    public class CommentAddRequestProfile : Profile
+    {
+        public CommentAddRequestProfile() 
```

Comment:
Request/Response also Dto postfix should be at the end not in the middle -> CommentAddProfileRequest